src/ray/core_worker/test/scheduling_queue_test.cc

```diff
@@ -48,6 +48,7 @@ TEST(SchedulingQueueTest, TestInOrder) {
   queue.Add(1, -1, fn_ok, fn_rej);
   queue.Add(2, -1, fn_ok, fn_rej);
   queue.Add(3, -1, fn_ok, fn_rej);
+  queue.ScheduleRequests();
```

Comment:
Sure! Because the new `SchedulingQueue:Add `function adds tasks to the queue but does not call `ScheduleRequests() ` to schedule their execution, we need now to call `ScheduleRequests()` manually in `scheduling_queue_test.cc `to let workers execute the tasks that we have added. The tests were written when `Add` would both enqueue tasks and schedule their execution, so the call to `queue.ScheduleRequests();` was previously not needed.